src/plugins/ncr/screens/node/selector.js

```diff
@@ -51,6 +53,7 @@ export default (state, ownProps, { schemas, formName, ...rest }) => {
     || updateNodeState.status === STATUS_PENDING;
 
   const isDeleteDisabled = !isDeleteGranted || isPending || nodeStatus.toString() === 'deleted';
+  const isForceSaveDisabled = isPending;
```

Comment:
@hunyoboy would want to account for isForceSaveGranted here, right?

Reply:
@efjacobson i don't think it's needed since it won't even get rendered if not granted to force save.